packages/web-storage/src/index.ts

```diff
@@ -124,6 +124,35 @@ export function setDataToStorage(key : string, data : any, storageType: string,
     localStorageInstance.set(bucketKey, data, expiresInMin);
   }
 }
+export function setDataToStorageStrict(key : string, data : any, storageType: string, expiresInMin = DEFAULT_STORAGE_EXPIRY_TIME, bucket = BUCKETS.OTHERS) {
```

Comment:
why any ?

Reply:
Why can we take an object and then de-structure it rather than adding more and more params.

Reply:
Any because we can't restrict the user to any specific data type. String, Number, Object, everything is possible. 

Secondly, adding a object would make sense if we have increasing params. If we add a object a parameter, will be tough for users to judge whats the parameter that they are entering. IDEs wont be able to help.

Reply:
But typescript if we are using then it will fine right. Its just code refactor, if we can do and nothing major needs to be updated in web_apps (If this is new function)